### PR TITLE
P2022-1943 add users to board -> P2022-2067 Simple BottomSheet

### PR DESCRIPTION
**Pull request by catchanus**

### CODE REVIEW

**Review by kmaslowiec — APPROVED** [review]
I think we can merge it now :)

**Comment by catchanus** [comment]
Well done:)

File: app/src/main/java/com/intive/patronage/retro/board/model/repo/BoardApi.kt [hunk 1]
```diff
@@ -5,10 +5,15 @@ import com.intive.patronage.retro.board.presentation.entity.Board
...
 
 interface BoardApi {
     @GET("api/v1/boards")
     suspend fun getBoards(): List<BoardRemote>
+
     @POST("api/v1/boards")
     suspend fun addBoard(@Body board: Board): BoardRemote
+
+    @POST("api/v1/boards/{id}/users")
+    suspend fun addUsers(@Path("id") id: Int, @Body users: List<String>): List<String>
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/board/model/repo/BoardApi.kt:18]: You are sure this URL is correct to the API?

**Comment by catchanus** [comment]
Good teamwork:)

File: app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt [hunk 2]
```diff
@@ -15,7 +15,7 @@ class MoreFragment : Fragment() {
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
-        savedInstanceState: Bundle?
+        savedInstanceState: Bundle?,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt:18]: This is necessary?

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt [hunk 3]
```diff
@@ -5,5 +5,5 @@ data class BoardCardsRemote(
     val cardText: String,
     val columnId: Int,
     val boardCardCreator: String,
-    val actionTexts: List<String>
+    val actionTexts: List<String>,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt:8]: I will repeat myself, this comma is needed here?

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt [hunk 4]
```diff
@@ -12,6 +12,5 @@ data class RetroRemote(
     val listColumns: List<ColumnsRemote>,
     @SerializedName("users")
     @Expose
-    val listUsers: List<UserRemote>
-
+    val listUsers: List<UserRemote>,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt:15]: I will repeat myself, this comma is needed here?

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt [hunk 5]
```diff
@@ -5,5 +5,5 @@ data class BoardCards(
     val cardText: String,
     val columnId: Int,
     val boardCardCreator: String,
-    val actionTexts: List<String>
+    val actionTexts: List<String>,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt:8]: I will repeat myself, this comma is needed here?

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt [hunk 6]
```diff
@@ -5,7 +5,7 @@ import com.intive.patronage.retro.retro.presentation.entity.Columns
 
 class RetroColumnsDiffUtil(
     private val oldColumnsList: List<Columns>,
-    private val newColumnsList: List<Columns>
+    private val newColumnsList: List<Columns>,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt:8]: I will repeat myself, this comma is needed here?

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt [hunk 7]
```diff
@@ -5,7 +5,7 @@ import com.intive.patronage.retro.retro.presentation.entity.RetroDetails
 
 class RetroDetailsDiffUtil(
     private val oldRetroDetailsList: List<RetroDetails>,
-    private val newRetroDetailsList: List<RetroDetails>
+    private val newRetroDetailsList: List<RetroDetails>,
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt:8]: I will repeat myself, this comma is needed here?

File: app/src/main/java/com/intive/patronage/retro/board/model/repo/BoardApi.kt (hunk 1 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/board/model/repo/BoardApi.kt:18]: This task is for assign users to the board. See URL on swagger.
https://intive-java-szczecin-2022.herokuapp.com/swagger-ui/index.html#/board-controller/assignUsersToBoard

File: app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt (hunk 2 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt:18]: It's not necessary of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt (hunk 3 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt:8]: It's not necessary of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt (hunk 4 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt:15]: It's not necessary of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt (hunk 6 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt:8]: It's not necessary of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt (hunk 7 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt:8]: It's not necessary of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt:8]: It's not needed of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt (hunk 6 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt:8]: It's not needed of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt (hunk 5 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt:8]: It's not needed of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt (hunk 3 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt:8]: It's not needed of course, but see ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt (hunk 2 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/more/presentation/view/MoreFragment.kt:18]: It's not necessary of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt (hunk 3 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/model/entity/BoardCardsRemote.kt:8]: It's not needed of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt (hunk 4 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/model/entity/RetroRemote.kt:15]: It's not needed of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt (hunk 5 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/entity/BoardCards.kt:8]: It's not needed of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt (hunk 6 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroColumnsDiffUtil.kt:8]: It's not needed of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt (hunk 7 above)
Comment (catchanus) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDetailsDiffUtil.kt:8]: It's not needed of course, but allowed. See ktlint's configuration file, which was made by us at the beginning of this project, especially line: ij_kotlin_allow_trailing_comma = true
We accepted, that the syntax/punctuation which went through our ktlint configuration is correct.

**Pull request merged**